sympy/functions/combinatorial/factorials.py

```diff
@@ -955,6 +955,8 @@ def eval(cls, n, k):
         if (k - 1).is_zero or ((n_nonneg or n_isint is False)
                 and (d - 1).is_zero):
             return n
+        if k == -1 and n == -1:
+            return S.One
```

Comment:
If we were going to make this change then it would be to evaluate for general negative integer values rather than just special casing these particular two values.

Reply:
I think there was a discussion regarding this, and `binomial(-1, -1)` and in general `binomial(kn, kn)` where `kn` is a negative value returning `0` change was brought by PR #14575. I've made a lot of tests where both `k` and `n` are negative evaluate to 1.

Reply:
I think `summation(binomial(2*k, k)/5**k, (k, -oo, oo))` is not evaluated since after this PR binomial gives 1 for negative numbers but `summation(binomial(2*k, k)/5**k, (k, 0, oo))` gives the correct answer equal to `sqrt(5)`. Should this test be changed too?

Reply:
I think it's better to discuss on the issue. There needs to be a decision on changing the behaviour of binomial before making a PR.

Reply:
Since the OP issue is closed, should i close this PR as well?